x-pack/plugins/discover_log_explorer/common/datasets/models/integration.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Dataset } from './dataset';
+import { IntegrationId, IntegrationType } from '../types';
+
+export class Integration {
+  id: IntegrationId;
+  name: IntegrationType['name'];
+  status: IntegrationType['status'];
+  version: IntegrationType['version'];
+  dataStreams: Dataset[];
```

Comment:
`dataStreams` is the property name coming from the API response, I kept it for that reason, but I agree it is not consistent. We can either rename it at this level when the entity model is applied, or maybe we could consider renaming it directly at the API level? wdyt?

Reply:
The API extracts that info from the data stream name pattern, so it would make sense to keep that name. Our client interprets it as a dataset, so it seems appropriate to rename it when applying the model. Is that reasonable?

Reply:
It sounds good yeah, I'll update its mapping to be consistently used as dataset